frontend/src/App.test.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+// This test has no actual use, it's only for future reference.
+// Should be deleted after actual tests are made.
+// test('renders learn react link', () => {
+//   render(<App />);
+//   const linkElement = screen.getByText(/learn react/i);
```

Comment:
Please remove files or lines of code that contain React logo or default values from the template

Reply:
Done